sandbox.config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "template": "node",
+  "infiniteLoopProtection": true,
+  "view": "terminal",
+  "container": {
+    "node": "14"
+  }
+}
```

Comment:
if you adding sandbox config, will be cool to make a demo link for fast testing your code)